bpf/cpu/cpu.bpf.c

```diff
@@ -94,6 +94,8 @@ struct unwinder_config_t {
   bool filter_processes;
   bool verbose_logging;
   bool mixed_stack_enabled;
+  bool python;
```

Comment:
nit: perhaps `python_enabled` and `ruby_enabled` could show the intent of these flags a bit better